src/redux/reducer.ts

```diff
@@ -1,7 +1,11 @@
 import { AnyAction } from 'redux'
 import { SET_LANGUAGE } from './constants'
 
-const initialState = {
+export interface Storei18nState {
```

Comment:
I think calling it something like `I18nState` or just plainly `StoreState` or `State` would be better. `Storei18nState` sounds wrong.